src/main/java/uk/gov/hmcts/reform/prl/controllers/citizen/RespondentSubmitResponseController.java

```diff
@@ -1,75 +0,0 @@
-package uk.gov.hmcts.reform.prl.controllers.citizen;
-
-import io.swagger.v3.oas.annotations.Operation;
-import lombok.extern.slf4j.Slf4j;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpHeaders;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestHeader;
-import org.springframework.web.bind.annotation.RestController;
-import uk.gov.hmcts.reform.prl.models.Element;
-import uk.gov.hmcts.reform.prl.models.complextypes.citizen.documents.UploadedDocuments;
-import uk.gov.hmcts.reform.prl.models.dto.ccd.CaseData;
-import uk.gov.hmcts.reform.prl.services.citizen.CaseService;
-import uk.gov.hmcts.reform.prl.services.document.DocumentGenService;
-
-import java.util.ArrayList;
-import java.util.List;
-import javax.validation.Valid;
-import javax.validation.constraints.NotNull;
-
-import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
-import static uk.gov.hmcts.reform.prl.constants.PrlAppsConstants.CITIZEN_UPLOADED_DOCUMENT;
-import static uk.gov.hmcts.reform.prl.utils.ElementUtils.element;
-
-@Slf4j
-@RestController
-public class RespondentSubmitResponseController {
-
-    @Autowired
```

Comment:
Why removed this file?

Reply:
Khurshed added it for C7 call back 
but we dont need it since i added for draft and final document endpoints
deleted after confirming with him